### PR TITLE
(chore) Removal of exporter components from the operator

### PR DESCRIPTION
**Pull request by rahulchheda**
Signed-off-by: Rahul M Chheda <rahul.chheda@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: pkg/controller/chaosengine/chaosengine_controller_test.go [hunk 1]
```diff
@@ -23,7 +23,6 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
```
Comment (ksatchit) [inline pkg/controller/chaosengine/chaosengine_controller_test.go:26]: are we sure about this

**Comment by ksatchit** [comment]
Why does the BCH drop a guideline here?

**Comment by ksatchit** [comment]
> Why does the BCH drop a guideline here?

Ok. Coverage! (thanks @ispeakc0de )

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ which provides bootstrap support for new operator projects, allowing teams to fo
 The Litmus Chaos Operator helps reconcile the state of the ChaosEngine, a custom resource that holds the chaos intent 
 specified by a developer/devops engineer against a particular stateless/stateful Kubernetes deployment. The operator performs
 specific actions upon CRUD of the ChaosEngine, its primary resource. The operator also defines secondary resources (the engine 
-runner pod and engine monitor service), which are created & managed by it in order to implement the reconcile functions. 
+runner pod), which are created & managed by it in order to implement the reconcile functions.
```
Comment (ksatchit) [inline README.md:37]: can we change this statement. It now has a single secondary resource

File: pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go [hunk 3]
```diff
@@ -32,7 +32,7 @@ type ChaosEngineSpec struct {
 	AnnotationCheck string `json:"annotationCheck,omitempty"`
 	//ChaosServiceAccount is the SvcAcc specified for chaos runner pods
 	ChaosServiceAccount string `json:"chaosServiceAccount"`
-	//Components contains the image of runnner and monitor pod
+	//Components contains the image of runnner
```
Comment (ksatchit) [inline pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go:35]: includes imagePullPolicy also with recent additions, right?

**Comment by ksatchit** [comment]
Looks good to me @rahulchheda . Have some minor comments/questions.

File: Makefile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Makefile for building Chaos Exporter
+# Makefile for building Chaos Operator
```
Comment (chandankumar4) [inline Makefile:1]: Check the line no:32 also

File: pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go (hunk 3 above)
Comment (chandankumar4) [reply pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go:35]: All these changes are for runner, Monitor only had image

File: pkg/controller/chaosengine/chaosengine_controller_test.go [hunk 5]
```diff
@@ -403,59 +285,3 @@ func TestGetChaosRunnerENV(t *testing.T) {
 		})
 	}
 }
-
-func TestGetChaosMonitorENV(t *testing.T) {
```
Comment (chandankumar4) [inline pkg/controller/chaosengine/chaosengine_controller_test.go:407]: Change the name in 166 line: I.e `Name:      "test-monitor",`

File: pkg/controller/watcher/watcher.go [hunk 6]
```diff
@@ -49,61 +41,6 @@ func WatchForRunnerPod(client client.Client, c controller.Controller) error {
...
-					return nil
-				}
-
-			}
-			return monitorPodRequest
-		}),
-	}
-	return handlerForMonitorPod
-}
-
```
Comment (chandankumar4) [inline pkg/controller/watcher/watcher.go:106]: Check the variable name in line no: 119

File: pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go (hunk 3 above)
Comment (ksatchit) [reply pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go:35]: Correct. Above comment says "Components contains the image of runner" I guess we can enhance the comment with these details also..

File: pkg/controller/chaosengine/chaosengine_controller_test.go (hunk 5 above)
Comment (rahulchheda) [reply pkg/controller/chaosengine/chaosengine_controller_test.go:407]: @chandankumar4 , its the name of chaosengine, we dont have to bother with that.

Comment (chandankumar4) [reply pkg/controller/chaosengine/chaosengine_controller_test.go:407]: yaa!! This name is also okay

**Pull request merged**